web/package.json

```diff
@@ -1,40 +1,32 @@
 {
-  "name": "gts",
+  "name": "web",
   "version": "0.1.0",
   "private": true,
-  "main": "src/index.tsx",
   "dependencies": {
-    "@equinor/eds-core-react": "^0.17.0",
-    "@equinor/eds-icons": "^0.10.0",
-    "@equinor/eds-tokens": "^0.7.0",
+    "@equinor/eds-core-react": "^0.20.3",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.3.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
Do these really need to be in production dependencies?

Reply:
They aren't.

Reply:
Doesn't need to, but it isn't a problem either: [https://github.com/facebook/create-react-app/issues/10219](url)